docs/site/tutorials/core/2-architecture.md

```diff
@@ -1,89 +1,96 @@
 for various parts to work together without knowing much about one another and
 introduces common patterns to solve such problems with example code:
 
-- Decoration and reflection: provide metadata management to capture knowledge of
-  various artifacts
-- Inversion of control and dependency injection: manage artifacts and their
+- **Decoration and reflection**: provide metadata management to capture
+  knowledge of various artifacts
+- **Inversion of control and dependency injection**: manage artifacts and their
   dependencies for resolution and composition - not the same old IoC and DI -
   it's built for async!
-- Extension points and extensions: organize artifacts with loose coupling and
-  promote extensibility
-- Chain of responsibility: align a set of action into a meaningful sequence or
-  flow
-- Observation and notification: enable (async) event driven collaboration
+- **Extension points and extensions**: organize artifacts with loose coupling
+  and promote extensibility
+- **Chain of responsibility**: align a set of action into a meaningful sequence
+  or flow
+- **Observation and notification**: enable (async) event driven collaboration
 
-## Overview
+## LoopBack: Helps you in building large scale Node.js applications
 
 LoopBack 4 core is an open source TypeScript platform for Node.js, specializing
 in building large scale applications with great flexibility, composability, and
 extensibility.
 
-## Key modules
+The following key modules are used by the LoopBack framework to offer API and
+Microservice capabilities. They can be used independently as a base platform to
+build large scale applications in TypeScript and Node.js.
 
 - @loopback/metadata
 - @loopback/context
 - @loopback/core
 
-These modules are used by the LoopBack framework to offer API and Microservice
-capabilities. They can be used independently as a base platform to build large
-scale applications in TypeScript and Node.js.
-
 ## Design goals
 
-Some characteristics of large scale applications:
+Some characteristics of large scale applications are:
 
 - Developed by many teams/developers
 - Maintained for many years and releases
 - Created as many modules
 - Have many dependencies
 
-Architectural disciplines are required to support such large scale applications.
+Architectural disciplines that are required to support such large scale
+applications, including:
 
 - Modular/Decoupling
 - Composable
 - Extensible
 
-Conflict of interest:
+There are conflict of interests between the two:
 
 - Modular/Decoupling/Consistent vs. Fragmentation/Friction
 - Composable vs. Extensible
 
 ## Scenario
 
-https://github.com/strongloop/loopback-next/tree/master/examples/greeter-extension
+We are going to use the
+[Greeter Extension](https://github.com/strongloop/loopback-next/tree/master/examples/greeter-extension)
+to illustrate how LoopBack can help you to build large scale Node.js
+applications.
 
 ## Series
```

Comment:
@raymondfeng, I'm a bit lost from here till the end of the file. Might need some more description from you in order to continue.